shared/js/background/startup.js

```diff
@@ -39,6 +42,25 @@ export async function onStartup () {
 
     Companies.buildFromStorage()
 
+    /**
+     * in Chrome only, try to initiate the `NewTabTrackerStats` feature
+     */
+    if (utils.getBrowserName() === 'chrome') {
```

Comment:
Can you confirm that this is false for edge and brave? What happens in chromium?

Reply:
Ok, tested Brave, it's `brave` there so we're good - I can test edge later too :)

Reply:
edge gives `edg` -> so it looks like we're all good here :)